### PR TITLE
Fix DHT/GPIO integration

### PR DESCRIPTION
**Pull request by Warukira**
## Description

Fix bug on DHT/GPIO expander integration.

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots
N/A
## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code

### CODE REVIEW

File: airrohr-firmware/DHT.cpp [hunk 1]
```diff
@@ -5,17 +5,18 @@ written by Adafruit Industries
 */
 
 #include "DHT.h"
+#include "PCF8574.h"
```
Comment (chegejames) [inline airrohr-firmware/DHT.cpp]: @Warukira is this necessary if it was included in DHT.h?

File: airrohr-firmware/DHT.cpp [hunk 2]
```diff
@@ -5,17 +5,18 @@ written by Adafruit Industries
...
-PCF8574 pcf8574; //class object
+PCF8574 pcf8574(0X02); //class object
 
 DHT::DHT(uint8_t pin, uint8_t type, uint8_t address) {
   _pin = pin;
   _type = type;
   pcf8574_address = address;
 
-  pcf8574.setAddress(address);
+  //pcf8574.setAddress(address);
```
Comment (chegejames) [inline airrohr-firmware/DHT.cpp]: @Warukira I am worried that we mixed up libs as the current PCF8574 has no such member class.

File: airrohr-firmware/PCF8574.cpp [hunk 3]
```diff
@@ -241,6 +241,10 @@ void PCF8574::begin()
     lastReadMillis = millis();
 }
 
+void PCF8574::setAddress(uint8_t address){
```
Comment (chegejames) [inline airrohr-firmware/PCF8574.cpp]: @Warukira did you add this as a patch, we should not be adding any extra functions to the core lib. This is already covered in the constructur.

File: airrohr-firmware/PCF8574.h [hunk 4]
```diff
@@ -120,6 +120,7 @@ class PCF8574
     PCF8574(TwoWire *pWire, uint8_t address, uint8_t sda, uint8_t scl, uint8_t interruptPin, void (*interruptFunction)());
 #endif
 
+    void setAddress(uint8_t address);
```
Comment (chegejames) [inline airrohr-firmware/PCF8574.h]: @Warukira we should not be adding methods not included in the original library

**Review by chegejames — CHANGES_REQUESTED** [review]

File: airrohr-firmware/PCF8574.h (hunk 4 above)
Comment (Warukira) [reply airrohr-firmware/PCF8574.h]: @chegejames ,okay. Fixing this.

File: airrohr-firmware/PCF8574.cpp (hunk 3 above)
Comment (Warukira) [reply airrohr-firmware/PCF8574.cpp]: Yes, I did @chegejames. Working on it.

File: airrohr-firmware/DHT.cpp (hunk 2 above)
Comment (Warukira) [reply airrohr-firmware/DHT.cpp]: @chegejames , that's why I commented it out.

File: airrohr-firmware/DHT.cpp (hunk 1 above)
Comment (Warukira) [reply airrohr-firmware/DHT.cpp]: @chegejames, I' m done. Pushing the requested changes, let me know if you have any more questions.

**Review by chegejames — APPROVED** [review]

**Pull request merged**